homebrew macvim build failed on macOS 10.14 with Xcode 10 beta

<pre>Last 15 lines from /Users/kaito/Library/Logs/Homebrew/macvim/02.make:
	from /System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/universal-darwin18/rbconfig.rb:214:in `<module:RbConfig>'
	from /System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/universal-darwin18/rbconfig.rb:7:in `<top (required)>'
	from /System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/rubygems.rb:9:in `require'
	from /System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/rubygems.rb:9:in `<top (required)>'
	from <internal:gem_prelude>:4:in `require'
	from <internal:gem_prelude>:4:in `<internal:gem_prelude>'

** BUILD FAILED **


The following build commands failed:
	CompileC /tmp/macvim-20180611-46855-pbvqpw/src/MacVim/build/MacVim.build/Release/MacVim.build/Objects-normal/x86_64/MMPreferenceController.o /tmp/macvim-20180611-46855-pbvqpw/src/MacVim/MMPreferenceController.m normal x86_64 objective-c com.apple.compilers.llvm.clang.1_0.compiler
(1 failure)
make[1]: *** [macvim] Error 65
make: *** [first] Error 2</pre>

--- a/src/MacVim/MMPreferenceController.h
+++ b/src/MacVim/MMPreferenceController.h
@@ -9,7 +9,7 @@
  */
 
 #import <Cocoa/Cocoa.h>
-#import <DBPrefsWindowController.h>
+#import "DBPrefsWindowController.h"
 
 @interface MMPreferenceController : DBPrefsWindowController {
     IBOutlet NSView *generalPreferences;

